extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -155,9 +155,16 @@ export class RemoteQueriesManager extends DisposableObject {
         queryItem.status = QueryStatus.Failed;
       }
     } else if (queryWorkflowResult.status === 'CompletedUnsuccessfully') {
-      queryItem.failureReason = queryWorkflowResult.error;
-      queryItem.status = QueryStatus.Failed;
-      void showAndLogErrorMessage(`Variant analysis execution failed. Error: ${queryWorkflowResult.error}`);
+      if (queryWorkflowResult.error?.includes('cancelled')) {
+        // workflow was cancelled on the server
+        queryItem.failureReason = 'Cancelled';
+        queryItem.status = QueryStatus.Failed;
+        void showAndLogErrorMessage('Variant analysis monitoring was cancelled');
```

Comment:
Minor: Is "monitoring" the right word here (and below on line 171)? It looks like the variant analysis (i.e. the workflow run) itself is cancelled 🐱
```suggestion
        void showAndLogErrorMessage('Variant analysis was cancelled');
```